src/main/java/no/cantara/realestate/mappingtable/repository/MappedIdRepositoryImpl.java

```diff
@@ -45,8 +44,12 @@ public List<MappedSensorId> find(MappingKey mappingKey) {
                 throw new MappingTableException("Bacnet not implemented");
             } else if (mappingKey.getKey() instanceof MetasysSensorId) {
                 throw new MappingTableException("Metasys not implemented");
-            } else if (mappingKey.getKey() instanceof EcoStruxureTrendSensorId) {
-                throw new MappingTableException("EcoStruxure not implemented");
+            } else if (mappingKey.getKey() instanceof String) {
+                matching = sensorIds.stream()
```

Comment:
This is a linear search, if the sensorIds list can grow big (more than 100), I would prefer to use a Map<Object, List<MappingKey>> sensorIdsByMappingKey or something similar, then applying the filters to the much smaller list in the map instead.